tests/apache-php/php-7.1/check_php_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+echo "Apache PHP module version should be 7"
+
+version=`apache2ctl -M | grep "[p]hp7_module"`
+
+if [ ! "$version" ]; then
```

Comment:
Not sure this would work, this is not PHP ;) If $version is empty, the expression will still be evaluated to true.

Reply:
If would directly use the grep expression with the if.